README.md

```diff
@@ -41,7 +41,8 @@ The description of the input parameters that can go in the run command is as fol
 8. **metamodel-components**: If this parameter is passed, then each metamodel component retreived from the vCenter server is saved in a different .json file under the metamodel directory.
 9. **host**: It is the IP Address of the host that serves the API. By default the value is < vcenter >
 10. **oas** : This parameter is used to specify as to which version of swagger file the user wants to generate. By default the generated files are of version 3 i.e openapi. If the user wants to generate the version 2 files, the parameter needs to be passed explicitly.
-11. **deprecate-slash-rest** : This parameter is used to deprecate the /rest APIs in the generated OpenAPI specification, only when the API to deprecate has an /api counterpart.
+11. **deprecate-slash-rest**: This parameter is used to deprecate the /rest APIs in the generated OpenAPI specification, only when the API to deprecate has an /api counterpart.
+12. **fetch-authentication-metadata**: Adds security information in the generated OpenAPI definitions. In order to do that it accesses API authentication metadata, which increases the processing time (~20-30 seconds).
```

Comment:
I meant (and said) rename the switch as well? Name it after the OpenAPI terminology for the "security" thing.

I assume when the switch is off we generate no "security" (which might be wrong, and we might be generating bad/broken/incomplete security) and when turned on we generate the good/correct "security"). If so, I would name it "generate-secutiry-schemes" or "add/produce-".

BTW, are these (this and the next one) the first boolean parameters for the tool? If so, how to we mark this for the user.

I just realized I'm commenting on a code comment here, but still I have these same concerns.